pkg/ebpf/tracer.go

```diff
@@ -166,6 +170,16 @@ func (t *Tracer) initPerfPolling() (*bpflib.PerfMap, error) {
 	return pm, nil
 }
 
+// setupExpvars setups up endpoint 8080 as the debug http endpoint and exposes some metrics for tracer
+func (t *Tracer) setupExpvars() {
+	go http.ListenAndServe(":8080", nil)
```

Comment:
Since the tracer runs as root, perhaps we don't want to expose this by default?  I think we may want to put this behind a debug flag.  Or send the data periodically to the main agent and expose it there?

Also, in a non-containerized environment, port 8080 is likely to be taken by a process on the customers machine, right ?